### ISSUE TITLE
Stables are underrepresented when shown separately across Polkadot's Parachains

### ISSUE THREAD

**Issue by joshuacheong** [DefiLlama/DefiLlama-Adapters#4677]
When looking at the section of stables [segmented by chains](https://defillama.com/stablecoins/chains), the values are significantly fragmented and do not represent the total stablecoin liquidity that exists with Polkadot ecosystem's Parachains.

Parachains that belong to the Polkadot ecosystem that exists on Defilama's dashboard today:
- Moonriver
- Moonbeam
- Astar
- Shiden
- Karura
- Acala (Numbers unrepresented)
- Crab

As of 18 Nov, this total amount in stables would amount to $90.44 Million and put our ecosystem at 17th in rank. Kindly provide a section within your representations to show that segmentation across the ecosystem.

**Comment by g1nt0ki** [comment]
@joshuacheong good point, probably need to create a new page for this, but atm the person working on this has left, so will keep this issue open and come back at some point to fix it
